### PR TITLE
fix(Space): replace native renderer with react renderer for web

### PR DESCRIPTION
**Pull request by krishnaacc123**
fix(space.web.test.js): Replace native render to react react render

### CODE REVIEW

**Comment by susheelachoudry** [comment]
lgtm

File: src/atoms/Space/__tests__/__snapshots__/Space.web.test.js.snap [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Renders <Space /> correctly should apply margin to child component 1`] = `<div />`;
```
Comment (kamleshchandnani) [inline src/atoms/Space/__tests__/__snapshots__/Space.web.test.js.snap]: There are no styles visible for paddings you applied. Also add some text. It's just rendering View right now

File: src/atoms/Space/__tests__/__snapshots__/Space.web.test.js.snap [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+exports[`Renders <Space /> correctly should apply padding to child component 1`] = `<div />`;
```
Comment (dhruvdutt) [inline src/atoms/Space/__tests__/__snapshots__/Space.web.test.js.snap]: Same as above. Add some text.

**Comment by deorukhkarchaitanya** [comment]
@krishnaacc123 @kamleshchandnani can we fix the snapshot tests here and get this merged? This is causing all PR checks to fail. :(

**Comment by kamleshchandnani** [comment]
@krishnaacc123 There's some major issue with the space component. It doesn't applies space. I didn't get much time to debug it. I'll merge it for now but please fix it and raise a new PR

**Comment by krishnaacc123** [comment]
@dhruvdutt Sorry couldn't work on this. As got engaged with some priority bugs.

**Comment by krishnaacc123** [comment]
> @krishnaacc123 There's some major issue with the space component. It doesn't applies space. I didn't get much time to debug it. I'll merge it for now but please fix it and raise a new PR

sure

**Comment by dhruvdutt** [comment]
No worries, please raise a PR with the fixes.

**Pull request merged**